code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -808,7 +808,7 @@
 	message_admins("[key_name_admin(usr)] [msg] in [ADMIN_VERBOSEJMP(specificTarget)].")
 	if (length(whoDyin))
 		for (var/mob/living/M in whoDyin)
-			admin_ticket_log(M, "[key_name_admin(usr)] [msg]")
+			admin_ticket_log(M, msg)
```

Comment:
I wouldn't do this, as this removes the user who launched the pod from the log.

Reply:
Not at my computer rn, but I'm pretty sure I'm grabbing usr in the New of the ticket log.

Reply:
It won't necessarily be the same `usr` who's launching the pod, though...

Reply:
`admin_ticket_log` doesn't pass a ckey, so `/datum/ticket_log/New` will set the `user` variable to `get_fancy_key(usr)` which is the same `usr` variable this call was passing to `key_name_admin`